src/TestSuite/Stub/ConsoleOutput.php

```diff
@@ -59,7 +59,7 @@ public function write($message, int $newlines = 1)
             $newlines--;
         }
 
-        return false;
+        return strlen($message);
```

Comment:
strlen() can operate on arrays?

Reply:
I overlooked the fact that $message can be array. Will fix it.

Returning the bytes written is necessary as it possible to overwrite a message in which case the length of previous msg is required to back track.